ports/qtbase/vcpkg.json

```
@@ -16,6 +16,14 @@
       "host": true,
       "default-features": false
     },
+    {
+      "name": "qtbase",
+      "default-features": false,
+      "features": [
+        "pcre2"
+      ],
+      "platform": "windows"
```

```suggestion
      "platform": "windows & static"
```
Your initial commit only did this for static builds? Are dynamic builds also affected?

Please also do this for double-conversion (from #22012) on windows, so the portfile is not changed twice just for that.

---

Due to I test dynamic build faild locally, I didn't add `static`. I double check dynamic builds, that faild reason is not related with feature `pcre2`. 
Done, thanks a lot.